admin-dev/themes/new-theme/js/pages/product/edit/product-options-manager.ts

```diff
@@ -79,6 +80,28 @@ export default class ProductOptionsManager {
     });
   }
 
+  private initProductVisibilityList(): void {
+    const defaultSelectedInput = document.querySelector(`${ProductMap.options.visibilityRadio}:checked`) as HTMLInputElement;
+    const descriptionField = document.querySelector<HTMLDivElement>('.js-description');
+
+    if (descriptionField === null) {
+      return;
+    }
+
+    descriptionField.innerHTML = `${defaultSelectedInput.dataset.description}`;
+
+    const inputs: NodeListOf<HTMLInputElement> = document.querySelectorAll(ProductMap.options.visibilityRadio);
+    inputs.forEach((input) => {
```

Comment:
```suggestion
    inputs.forEach((input: HTMLInputElement) => {
```
This is optional, since TypeScript is supposed to infer the type correctly